_docs/collection-details.md

```diff
@@ -183,11 +200,112 @@ Once the transaction is created, instructions for completing payment will be sen
 
 Once the funds have been successfully received from the sender, `payin_method.paid_in` and `transaction.paid_in` webhooks will be sent out.
 
+# XOF mobile collections
+
+To initiate a XOF mobile collection, please use the following details (`phone_number` used below are examples):
+At the moment, we only support collections in:
+
+- Senegal(Orange) - this requires the sender to request an OTP auth code via USSD #144#391\*ORANGE_MONEY_PIN_CODE#. The auth code retrieved should be included in the `otp` parameter in `in_details`.
+
+{% capture data-raw %}
+
+```javascript
+"input_currency": "XOF",
+"payin_methods": [
+  {
+    "type": "XOF::Mobile",
+    "ux_flow": "ussd_voucher",
+    "in_details": {
+      "phone_number": "+221123456789", // In international format
+      "mobile_provider": "orange",
+      "otp": "123456"
+    }
+  }
+],
+```
+
+{% endcapture %}
+
+{% include language-tabbar.html prefix="collection-xof-mobile" raw=data-raw %}
+
+If the OTP matches the auth code that the sender got, the collection process starts:
+{% capture data-raw %}
+
+```javascript
+{
+  id: "9b25ca43-1812-46b2-ae0c-57acefec0a34",
+  type: "XOF::Mobile",
+  ux_flow: "ussd_voucher",
+  state: "pending",
+  state_reason_details: {
+    code: 14,
+    category: "pending",
+    messages: [
+      "Pending",
+      "Pending status update"
+    ],
+    description: "This transaction is awaiting status update from provider."
+  },
+  in_details: {
+    phone_number: "+221123456789",
+    mobile_provider: "orange",
+    otp: "123456"
+  },
+  out_details: {
+    "style": "info"
+  },
+  instructions: {
```

Comment:
@evanson shouldn't we add proper instructions here and in `out_details` as well?

Reply:
The instructions are necessary **before** the API call and would therefore be redundant.

Reply:
working on them